### PR TITLE
test: some preparation for live test run

### PR DESCRIPTION
**Pull request by yugangw-msft**
1. Display start and end time
2. Trim line breaks so external test file can load

### CODE REVIEW

File: test/testlistarmlive.txt [hunk 1]
```diff
@@ -11,7 +11,6 @@ commands/arm/role/arm.role.assignment.audit-tests.js
 commands/arm/ad/arm.ad-tests.js
 commands/arm/ad/arm.ad-app-tests.js
 commands/arm/ad/adUtils-test.js
-commands/arm/ad/role/roleAssignments-tests.js
```
Comment (yugangw-msft) [inline test/testlistarmlive.txt]: this file path no longer exists

**Comment by yugangw-msft** [comment]
@huangpf , please review

File: scripts/unit.js [hunk 2]
```diff
@@ -69,4 +70,12 @@ args.push(reporter);
 //for clean shutdown of the event loop, so silly log gets drained to disk.
 args.push('--no-exit');
 
+console.log("Start:" + new Date());
+
+//console.log(JSON.stringify(args, null, 2));
```
Comment (huangpf) [inline scripts/unit.js]: ?

**Comment by huangpf** [comment]
LGTM.

**Pull request merged**